tools/toolchains/BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+load("@rules_proto//proto:defs.bzl", "proto_lang_toolchain")
+
+proto_lang_toolchain(
```

Comment:
I'm looking through the related issues trying to understand why we need to bring in the Java implementation separately while the rest of it arrives automatically from `toolchains_protoc`. Are we waiting for some work to happen in the Java Protobufs implementation, and have to work around in the meantime?